lib/db.php

```diff
@@ -143,6 +143,7 @@ static public function connect($site_config) {
             return self::$dbh[$key];
         } catch (Exception $oops) {
             Hm_Debug::add($oops->getMessage());
+            Hm_Msgs::add('ERRUnable to connect to the database. Please check your configuration settings and try again.');
```

Comment:
There can be several reasons and not necessarily the configuration issue. I feel like this error message only addresses one case.

Reply:
Any thought @kroky ?

Reply:
Since the debugging msg contains the actual error, I think we can have a relatively short user facing message that database connection cannot be established and site/app must be configured correctly before proceeding. The admin configuring the app will know where to look to find the debugging message and fix the issue.